ceph.spec.in

```diff
@@ -152,8 +152,13 @@ BuildRequires:	fuse-devel
 # enough.
 BuildRequires:	devtoolset-8-gcc-c++ >= 8.3.1-3.1
 %else
+%if 0%{?sle_version} == 150200
+# gcc-c++ is version 7 which does not work with "-DWITH_SEASTAR=ON"
+BuildRequires:	gcc9-c++
```

Comment:
gcc8 would suffice. but the higher the merrier ! if you have concern on this dependency, probably you could guard it with
```
%if 0%{with seastar}

%endif
```
? as crimson is currently only at its experimental stage. i am afraid that adding this dependency would hurt (or even slow down the build of) ceph build of SLE in some way.

Reply:
Done